app/views/hub/state_file/efile_submissions/index.html.erb

```diff
@@ -57,8 +57,10 @@
             <%= link_to submission.current_state.humanize(capitalize: false), hub_state_file_efile_submission_path(id: submission.id), class: "underline" %>
           </td>
           <td class="index-table__cell">
-            <% submission.last_transition&.efile_errors&.each do |error| %>
-                <span class="tooltip max-250 error-<%= error.id %>" data-position="left" title="<%= error.message %>">
+            <% errors = submission.last_transition&.efile_errors  %>
+            <% errors = submission&.efile_submission_transitions&.where(to_state: 'rejected')&.last&.efile_errors if ['waiting', 'notified_of_rejection'].include?(submission.current_state) %>
+            <% errors&.each do |error| %>
```

Comment:
For for readability sakes:

```
<% if ['waiting', 'notified_of_rejection'].include?(submission.current_state)
    errors = submission&.efile_submission_transitions&.where(to_state: 'rejected')&.last&.efile_errors 
```suggestion
            <% errors = submission&.efile_submission_transitions&.where(to_state: 'rejected')&.last&.efile_errors if ['waiting', 'notified_of_rejection'].include?(submission.current_state) %>
            <% errors&.each do |error| %>
```
end
%>

```